airsonic-main/src/main/java/org/airsonic/player/security/GlobalSecurityConfig.java

```diff
@@ -125,6 +125,22 @@ protected void configure(HttpSecurity http) throws Exception {
             restAuthenticationFilter.setEventPublisher(eventPublisher);
             http = http.addFilterBefore(restAuthenticationFilter, UsernamePasswordAuthenticationFilter.class);
 
+            // Try to load the 'remember me' key from command-line properties...
+            String rememberMeKey = System.getProperty("airsonic.rememberMeKey");
```

Comment:
Shouldn't this be in the settingsservice instead?

Reply:
Done.